docs/developers/guides/run-a-node/erigon.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: Erigon
+description: Install the Erigon client to run a Linea node.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import VolumeCreation from "./volume-creation.mdx";
+
+Erigon is a client implementation focused on performance and saving disk space, written in Go.
+
+:::info important
+Install and run an Erigon if you want to follow the Linea network by 
+maintaining a local copy of the blockchain. However, if you want to interact with the network and use
+Linea-specific methods and features, you should [install Linea Besu](./linea-besu.mdx) instead.
+:::
+
+You can run Erigon from a [binary distribution](#run-using-the-binary-distribution) or [using Docker](#run-using-docker).
+
+## Run using the binary distribution
+
+### Step 1. Install Erigon
+
+[Download and install the Erigon client](https://erigon.gitbook.io/erigon/basic-usage/getting-started).
```

Comment:
I'm hesitant to add info specific to third-party tools/apps, because it's hard to maintain if anything changes on their end.

However, it may be worth adding an admonition telling users to ensure they review Erigons software requirements documentation.